app/services/job-state.js

```diff
@@ -18,7 +23,8 @@ export default Ember.Service.extend({
     const result = yield this.get('store').filter(
```

Comment:
FUNNY that I can comment on this line but not the one above because it’s in the fold, but this one isn’t part of the PR either!

Another question for you, @drogus; why is it that `queued` is considered `running`? The labels of the sections are running and queued, seems confusing…???

Reply:
Good question! I would say that `queued` state shouldn't be there. I tracked it down to [this commit](https://github.com/travis-ci/travis-web/commit/c0917d2), but we will have to confirm if this was done by mistake or is it a deliberate change for some reason that we're not aware of.

Reply:
I couldn't find any rationale behind that change, so I think that we should change `queued` to be only for queued jobs list.